lib/matplotlib/__init__.py

```diff
@@ -605,6 +607,7 @@ def _create_tmp_config_dir():
     """
     configdir = os.environ['MPLCONFIGDIR'] = (
         tempfile.mkdtemp(prefix='matplotlib-'))
+    atexit.register(shutil.rmtree, configdir)
```

Comment:
Won't this rm the config directory even if it isn't a temporary directory?

Reply:
it's in "_create_tmp_config_dir"...